packages/postgres-queries-builder/src/helpers/sanitizeIdentifier.spec.ts

```diff
@@ -0,0 +1,50 @@
+import sanitizeIdentifier from './sanitizeIdentifier';
```

Comment:
Are test files named `.test.` or `.spec.`?
Also, I'm in favour of adopting [KebabCase](http://wiki.c2.com/?KebabCase) for file names.